integration-tests/README.md

```diff
@@ -31,9 +31,24 @@ import { ALITH_ADDRESS } from "@moonwall/util";
 
 ### Usage Examples (non-exhaustive)
 
+You should have installed `python` for using `sqlite3` and then used `pnpm rebuild && pnpm rebuild sqlite3`.
+
+Run `pnpm install` in the `integration-tests` folder before running any of the following commands.
+
+Useful for integration testing:
+
+- `./integration-tests/scripts/deploy-zombienet.sh` - Deploy a local relay-parachain network for zombienet.
+- `pnpm chopsticks xcm -r polkadot -p ./configs/hydradx.yml -p ./configs/zeitgeist.yml` - Deploy a local relay-parachain fork network via chopsticks to test XCM.
+- `./integration-tests/scripts/deploy-zombienet.sh --test` - Run ZNDSL zombienet tests on a local relay-parachain network.
+- `pnpm exec moonwall test zombienet_zeitgeist_upgrade` - Test Zeitgeist runtime upgrade on zombienet for the local network.
```

Comment:
This often takes a little bit longer. It's often on my machine too: `fetching metrics`...

Reply:
So the fact that it says "Error" shouldn't concern me?

Reply:
I can't do anything about it. It is just stated, because the nodes are not ready to fetch the metrics yet. It's because the nodes are still in the process of starting the metric endpoint.

Reply:
Okay, we should add this to the README

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1194/commits/6e574ff3536bab823e47f4bebbb124232de521b2